res/values/strings.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="app_name" translatable="false">Aard 2</string>
+    <string name="app_name" translatable="false">Aard 2D</string>
```

Comment:
Please remove this modification. I've not yet planned on altering the app label.